Battleships/Program.cs

```diff
@@ -0,0 +1,343 @@
+﻿    
+Grid[] playerGrid = new Grid[2];
+int[] shipSizes = new int[5] {5, 4, 3, 3, 2};
+Ship[] p1ships = new Ship[shipSizes.Length];
+Ship[] p2ships = new Ship[shipSizes.Length];
+int rows = 10;
+int collumns = 10;
+ConsoleColor[] playerColor = new ConsoleColor[2] { ConsoleColor.Yellow , ConsoleColor.Blue};
+
+for (int p = 0; p < 2; p++)
+{
+    playerGrid[p] = new Grid(rows, collumns);
+    playerGrid[p].position = new GridPosition[rows, collumns];
+    for (int row = 0; row < rows; row++)
+    {
+        for (int collumn = 0; collumn < collumns; collumn++)
+        {
+            playerGrid[p].position[row, collumn] = new GridPosition(" ");
```

Comment:
Consider making the symbols into enums and the printing logic in the class. e.g.

```csharp
public class GridPosition{
   public Ship ship;
   public string GetSymbol(){
      if(ship == null) return " ";
      // etc.
   }
}
```